javav2/example_code/acmpca/src/main/java/com/example/acmpca/UntagCertificateAuthority.java

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.example.acmpca;
+
+import java.util.ArrayList;
+import java.util.List;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.acmpca.AcmPcaClient;
+import software.amazon.awssdk.services.acmpca.model.AcmPcaException;
+import software.amazon.awssdk.services.acmpca.model.Tag;
+import software.amazon.awssdk.services.acmpca.model.UntagCertificateAuthorityRequest;
+
+// snippet-start:[acmpca.java2.UntagCertificateAuthority.main]
+/**
+ * Before running this Java V2 code example, set up your development 
+ * environment, including your credentials.
+ * 
+ * For more information, see the following documentation topic:
+ * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ */
+public class UntagCertificateAuthority {
+
+  public static void main(String[] args) throws Exception {
+
+    final String usage = 
+        """
+           Usage: <region> <caArn> <tagRemoval>
+
+           Where:
+               region - The AWS region (e.g. us-east-1)
+               caArn - The ARN of the certificate authority
+               tagRemoval - The tag(s) to remove. Specify either 'KeyOnly' or 'Key&Value'
```

Comment:
I don't think you need this to be parameterized; for the purpose of the sample I think you can just (attempt to) remove both types (both key-only and key-value) of tags in the same API call